src/contracts/interfaces/IEigenPod.sol

```diff
@@ -49,6 +49,15 @@ interface IEigenPod {
         int256 sharesDeltaGwei;
     }
 
+    struct VerifiedPartialWithdrawalBatch{
+        // amount being proven for withdrawal
+        uint64 provenPartialWithdrawalSumGwei;
+        // the latest timestamp proven until
+        uint64 mostRecentWithdrawalTimestamp;
+        // upper bound of the withdrawal period
+        uint64 endTimestamp;
```

Comment:
these 2 fields seem redundant, and I think(?) I'm not seeing the `mostRecentWithdrawalTimestamp` field being used in the EigenPod contract, at least. Please correct me if I'm wrong

Reply:
check line 442 - we're basically checking that the offchain proven mostRecentWithdrawalTimestamp == mostRecentWithdrawalTimestamp in the contract

Reply:
ahhhhhh OK nevermind then, resolving.